tracing-fmt/src/format.rs

```diff
@@ -152,6 +159,11 @@ where
         writer: &mut dyn fmt::Write,
         event: &Event,
     ) -> fmt::Result {
+        #[cfg(feature = "tracing-log")]
+        let normalized_meta = event.normalized_metadata();
+        #[cfg(feature = "tracing-log")]
+        let meta = normalized_meta.as_ref().unwrap_or_else(|| event.metadata());
+        #[cfg(not(feature = "tracing-log"))]
```

Comment:
It's a shame that this is repeated, rather than factored out into a function. But, I guess this is necessary since we can't return a reference to the normalized metadata from a function, since the reference would outlive the local, so I suppose this is necessary.